qemu/qemu.go

```diff
@@ -824,8 +828,13 @@ func (vfioDev VFIODevice) Valid() bool {
 // QemuParams returns the qemu parameters built out of this vfio device.
 func (vfioDev VFIODevice) QemuParams(config *Config) []string {
 	var qemuParams []string
+	var deviceParam string
 
-	deviceParam := fmt.Sprintf("vfio-pci,host=%s", vfioDev.BDF)
+	if runtime.GOARCH != "s390x" {
+		deviceParam = fmt.Sprintf("vfio-pci,host=%s", vfioDev.BDF)
```

Comment:
I think this `if..else` condition is not needed since you can use the variable `vfio`, right?

Reply:
No vfio is not an available device

Reply:
I guess what @devimc meant was something like:
```go
deviceParam = fmt.Sprintf(Vfio+",host=%s", vfioDev.BDF)
```
and removing the if-else

Reply:
sorry I don't understand why you can't do something like this

```go
deviceParam = fmt.Sprintf("%s,host=%s", vfio, vfioDev.BDF)
```

Reply:
might be I missed something, please help me to understand

Reply:
Other devices like virtio-net are alias to virtio-net-ccw on Z and  virtio-net-pci on others. However, a similar alias for vfio does not exist. Hence this needs to be kept as is.

Reply:
So just writing 'vfio' would result in an error

Reply:
> However, a similar alias for vfio does not exist

what do you mean? 

At the beginning of the file you are declaring `Vfio = "vfio-pci"` but you change its value at `init()` if `runtime.GOARCH == "s390x"`, I wonder if is possible to use the`Vfio` variable or declare a new variable (like you did with `VirtioSCSI`) in order to rid of this `if...else` condition.

Reply:
let me update my example 

```go
deviceParam = fmt.Sprintf("%s,host=%s", Vfio, vfioDev.BDF)
```

Reply:
yup that's possible... I thought you were asking me to change vfio-pci to generic vfio

Reply:
@devimc Updated